Source: Martin von Gagern
        http://forum.openscenegraph.org/viewtopic.php?p=44685#44685
Upstream: accepted (in trunk)
Reason: support for xine-lib-1.2+

--- OpenSceneGraph-3.0.1/src/osgPlugins/xine/video_out_rgb.c.orig	2013-05-06 13:57:51.871181504 +0300
+++ OpenSceneGraph-3.0.1/src/osgPlugins/xine/video_out_rgb.c	2013-05-06 14:00:18.845173168 +0300
@@ -2769,8 +2769,13 @@
 	clear(rgb_class, sizeof(rgbout_class_t));
 
 	rgb_class->driver_class.open_plugin     = open_plugin;
+#if XINE_MAJOR_VERSION < 1 || (XINE_MAJOR_VERSION == 1 && XINE_MINOR_VERSION < 2)
 	rgb_class->driver_class.get_identifier  = get_identifier;
 	rgb_class->driver_class.get_description = get_description;
+#else
+	rgb_class->driver_class.identifier  = get_identifier(NULL);
+	rgb_class->driver_class.description = get_description(NULL);
+#endif
 	rgb_class->driver_class.dispose         = dispose_class;
 
 	return(rgb_class);
